be/src/Project/Web/Seller/Seller.Web/Areas/Media/ApiRequestModels/UploadMediaItemRequestModel.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Seller.Web.Areas.Media.ApiRequestModels
 {
-    public class UploadMediaRequestModel : RequestModelBase
+    public class UploadMediaItemRequestModel : RequestModelBase
```

Comment:
**suggestion:** it's too much of a change. We should only get rid of the 'Medias'. I'd leave everything else as it was.